admin/server/reports.go

```diff
@@ -283,7 +265,23 @@ func (s *Server) UnsubscribeReport(ctx context.Context, req *adminv1.Unsubscribe
 			return nil, status.Errorf(codes.InvalidArgument, "failed to find report token: %s", err.Error())
 		}
 
-		userEmail = reportTkn.RecipientEmail
+		if reportTkn.RecipientEmail == "" {
```

Comment:
Is it easy to break it out by notifier type? If not, I think it's fine to unsubscribe both places because this will probably never happen in practice.

Reply:
done